ethereum/core/src/main/java/org/hyperledger/besu/ethereum/storage/keyvalue/KeyValueStoragePrefixedKeyBlockchainStorage.java

```diff
@@ -72,6 +74,11 @@ public KeyValueStoragePrefixedKeyBlockchainStorage(
         .orElse(Lists.newArrayList());
   }
 
+  @Override
+  public Optional<Hash> getFinalized() {
+    return get(CONSTANTS_PREFIX, FINALIZED_BLOCK_HASH_KEY).map(this::bytesToHash);
```

Comment:
I don't think there is any benefit to re-using the CONSTANTS_PREFIX.  I don't think it will hurt necessarily, but it might be clearer if we added a CONSENSUS_PREFIX.

Reply:
I agree that the name CONSTANTS_PREFIX is wrong, because every value in it is actually changing, so I am renaming it to VARIABLES_PREFIX